content/episodes/2019-07-29-S05E04-tech-worker-resistance.md

```diff
@@ -0,0 +1,164 @@
+
+    Okay. Now over to Amazon. That's where staff and shareholders recently spoke out against the world's biggest online retailer, claiming that the company has weak environmental policies. While Amazon does invest in some renewable energy products and has committed to reducing the carbon footprint of its trucking fleet, some say the company must do much more.
+
+    **Rebecca:** Climate scientists have been telling us for decades that there is one very easy number to remember of what the right number is for a fossil fuel extraction, and that is zero.
+
+    **Manoush:** Rebecca Sheppard works at Amazon.
+
+    **Rebecca:** We cannot continue to expand and accelerate the extraction of fossil fuels and the corresponding carbon emissions, and yet Amazon web services has continued to partner with those fossil fuel services, which is extremely alarming and doesn't show the commitment to sustainability and innovation that you would expect from Amazon.
+
+    **Manoush:** Amazon Web Services is a massive network of data centers that support countless companies. Rebecca thinks an innovative company like Amazon could be a leader by making those data centers more environmentally responsible. She's not the only one. In May, shareholders asked the company's board of directors to come up with a robust climate change plan and 8,000 employees signed a public letter calling for action. Rebecca was one of them and it gave her hope.
+
+    **Rebecca:** It's incredible that at the place that you work when you start to feel down, rather than feeling small or disempowered that you can actually do something in your current company, to make the change that you know is right. You don't have to be on the wrong side of history just because where you currently find yourself. You can work to make the company or organization you're currently identified with better.
+
+    **Manoush:** Ultimately the shareholder resolution was voted down. Amazon says it's already working on reducing emissions, but it's now agreed to make information about its carbon footprint public for the first time. As for Rebecca and the other members of the Amazon employees for climate justice, they say they are not letting up and they have another shareholder resolution planned for next year.
+
+    **Rebecca:** I think the best form of company loyalty and ensuring you're getting the most from the place you work is to challenge your company to be better.
+
+    **Manoush:** By the way, if you missed our previous episode, it was all about the internet's carbon footprint, so go check it out. It's really interesting. Uber, Google, Amazon. Those are just a few examples of companies facing new found tech resistance from within. And this activism is growing, but what's less clear is: why now?
+
+    **Cind:** I think that many tech workers were lured in with the idea that they were making the world a better place.
```

Comment:
This name is also oddly truncated, and should either be "Cindy" or the full name "Cindy Cohn"